ql/src/java/org/apache/hadoop/hive/ql/exec/tez/HiveSplitGenerator.java

```diff
@@ -444,4 +572,12 @@ private AvailableSlotsCalculator getAvailableSlotsCalculator() throws Exception
     slotsCalculator.initialize(conf, this);
     return slotsCalculator;
   }
+
+  /**
+   * Convenience method for callers that want to disable the serialization of splits to filesystem.
+   */
+  public HiveSplitGenerator splitFsSerialization(boolean splitFsSerialization) {
```

Comment:
wait, input param is not used at all

Reply:
LOL, right, I just used this for disabling :D
I'll fix this to:
{code}
  public HiveSplitGenerator splitFsSerialization(boolean splitFsSerialization) {
    HiveConf.setIntVar(jobConf, HiveConf.ConfVars.HIVE_TEZ_SPLIT_FS_SERIALIZATION_THRESHOLD,
        splitFsSerialization ? HiveConf.ConfVars.HIVE_TEZ_SPLIT_FS_SERIALIZATION_THRESHOLD.getDefaultValue() : -1);
    return this;
  }
{code}

Reply:
regarding:
```
also, why do you need this method at all, since you pass the jobConf to HiveSplitGenerator constructor? pass conf with disabled serialization
```
because I was assuming this cleaner and more convenient, so the caller doesn't have to be aware of the config details